File: module_video/src/main/java/com/yyxnb/module_video/VideoActivity.java
package com.yyxnb.module_video;

import android.support.v4.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.module_base.base.ContainerActivity;
import com.yyxnb.module_video.fragments.VideoMainFragment;

import static com.yyxnb.module_base.arouter.ARouterConstant.VIDEO_VIDEO;

@Route(path = VIDEO_VIDEO)
public class VideoActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new VideoMainFragment();
    }

}

File: module_video/src/main/java/com/yyxnb/module_video/VideoActivity.kt
package com.yyxnb.module_video

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.alibaba.android.arouter.facade.annotation.Route

import com.yyxnb.module_base.arouter.ARouterConstant.VIDEO_VIDEO
import com.yyxnb.module_video.fragments.VideoMainFragment

@Route(path = VIDEO_VIDEO)
class VideoActivity : AppCompatActivity() {

    fun getContainerViewId():Int = R.id.atyContent

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_video)
    }
}
